### PR TITLE
Make scanner from scannable easier to use

### PR DESCRIPTION
**Pull request by kevintok**
Add more ore colors
Add more fluid colors
Convert some ores to common ores
Change duration of highlight to 30s

### CODE REVIEW

File: config/scannable-client.toml [hunk 1]
```diff
@@ -5,11 +5,39 @@
...
+  "c:zinc_ores=0xbac4c8",
+]
 #The colors for fluids used when rendering their result bounding box
 #by fluid name. See `blockColors` for format entries have to be in.
 fluidColors = []
 #The colors for fluids used when rendering their result bounding box
 #by fluid tag. See `blockColors` for format entries have to be in.
-fluidTagColors = ["minecraft:lava=0x9f5224", "minecraft:water=0x4040ff"]
-
+fluidTagColors = ["minecraft:lava=0x9f5224", "minecraft:water=0x4040ff", "c:tea=0xbc7b3b", "c:oil=373a36"]
```
Comment (TheStaticVoid) [inline config/scannable-client.toml:43]: Oil's color is not in hexadecimal format. Crashes on startup with this

File: config/scannable-common.toml [hunk 2]
```diff
@@ -56,12 +56,26 @@
 [ores]
 	#Registry names of blocks considered 'common ores', requiring the common ore scanner module.
 	commonOreBlocks = ["minecraft:clay"]
```
Comment (TheStaticVoid) [inline config/scannable-common.toml:58]: Why is clay considered an ore?

**Comment by TheStaticVoid** [comment]
image
I'm using your config changes and a lot of the ores still look the same color?

File: config/scannable-common.toml (hunk 2 above)
Comment (kevintok) [reply config/scannable-common.toml:58]: It is kinda hard to look for in vanilla lol. I can remove em if you want it

File: config/scannable-client.toml (hunk 1 above)
Comment (kevintok) [reply config/scannable-client.toml:43]: Oh oops, missed the 0x. I'll update

**Comment by kevintok** [comment]
Can you identify which those are? I'm using the values in https://github.com/TheStaticVoid/StaTech-Industry/blob/main/kubejs/exported/tags/minecraft/item.txt
but not all of them. I did skip some tech reborn one, I can go back and check.

**Comment by kevintok** [comment]
Updated and tested. To note, a lot of ores are white (salt, silver, galena), and the one colored black(coal, lignite coal) shows up as white as well.

**Comment by TheStaticVoid** [comment]
image
The following ores do not have color and should be updated to. Please assume deepslate variants are included in the list:
- Tin Ore
- Titanium Ore
- Tungsten Ore
- Uranium Ore
- Bauxite Ore
- Lead Ore
- Lignite Coal Ore
- Nickel Ore
- Topaz Ore
- Amethyst Ore
- Citrine Ore
- Onyx Ore
- Shimmerstone Ore
- Azurite Ore
- Stratine Ore

Each of these ores have very distinct colors and the highlight color should attempt to match that color.

Other than those ones, the rest look very good!

**Comment by TheStaticVoid** [comment]
Thank you, I will review this when I have the chance tomorrow. If everything is in order I will aim to add it for the 1.0.4 release

**Comment by kevintok** [comment]
Fixed some more ores. Btw, I found an issue with tech reborn ores. They don't show up on scanner, even though I've tried adding `forge:ores` and `c:ores` tag manually.

**Comment by TheStaticVoid** [comment]
That's fine, most TR ores are not in world gen. I changed the target to the 1.0.4 branch, everything looks good.

**Pull request merged**